Document the instruction how to build on mac

Personally, I couldn't build grv on macOS. I have **ncurses** and **readline**, but they seem to be incompatible or smth, because building process fails with issues like this:
```
cmd/grv/readline.go:200:30: could not determine kind of name for C.rl_display_prompt

clang errors for preamble:
cmd/grv/readline.go:14:2: error: use of undeclared identifier 'rl_catch_signals'
        rl_catch_signals = 0;
```

--- a/cmd/grv/readline.go
+++ b/cmd/grv/readline.go
@@ -1,7 +1,7 @@
 package main
 
-// #cgo darwin CFLAGS: -I/usr/local/Cellar/readline/7.0/include
-// #cgo darwin LDFLAGS: -L/usr/local/Cellar/readline/7.0/lib
+// #cgo darwin CFLAGS: -I/usr/local/opt/readline/include
+// #cgo darwin LDFLAGS: -L/usr/local/opt/readline/lib
 // #cgo LDFLAGS: -lreadline -lhistory
 //
 // // TODO: Find way of determining paths above in Makefile and providing them

